spec/mailers/user_mailer_spec.rb

```diff
@@ -97,6 +97,26 @@
     end
   end
 
+  describe "Review update" do
+    let(:invitee) { create(:user) }
+    let(:user) { create(:user) }
+    let(:ac) { create(:associate_consultant, user: user) }
+    let(:review) { create(:review, associate_consultant: ac) }
+    let(:mail) { UserMailer.review_update(invitee, review) }
+
+    it 'includes the feedback deadline' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.